views/editprofile.view.php

```diff
@@ -4,72 +4,59 @@
 require $_SERVER['DOCUMENT_ROOT'] . '/views/partials/header.php';
 require $_SERVER['DOCUMENT_ROOT'] . '/views/partials/nav.php';
 ?>
-<form action="/controllers/edit_profile.php" method="POST">
-    <label for="username">Username</label>
-    <input type="text" name="username" id="username" value="<?= $user['username'] ?>">
-    <label for="email">Email</label>
-    <input type="text" name="email" id="email" value="<?= $user['email'] ?>">
-    <label for="password">Password</label>
-    <input type="password" name="password" id="password" value="<?= $user['password'] ?>">
-    <label for="confirm_password">Confirm Password</label>
-    <input type="password" name="confirm_password" id="confirm_password" value="<?= $user['password'] ?>">
-    <label for="isAdmin">Admin</label>
-    <input type="checkbox" name="isAdmin" id="isAdmin" value="<?= $user['isAdmin'] ?>">
-    <input type="submit" value="Submit">
-</form>
-<?php require $_SERVER['DOCUMENT_ROOT'] . '/views/partials/footer.php'; ?>
+<!-- input fields and edit buttons for all categories below -->
 
-
-    
-
-    <main>
-    <!-- input fields and edit buttons for all categories below -->
+<section>
     <h1> Edit Profile</h1>
-    
-    <section>
-    <h1>Personal Information</h1> 
-    <form action="/placeholder">
-    <label for="email">First name:</label>
-    <input type="text" placeholder="John"> <br>
-    <label for="wachtwoord">Last name:</label>
-    <input type="text" placeholder="Van den Berg"> <br>
-<form action="/placeholder">
-<label for="Voornaam">Email:</label>
-<input type="text" Placeholder="JohnvanDenBerg@gmail.com"> <br>
-<label for="Achternaam">Password:</label>
-<input type="text" placeholder="**********"> <br>
-<label for="Achternaam"> Confirm Password:</label>
-<input type="text" Placeholder="**********"> <br>
-<form action="/placeholder">
-<textarea name="" id="" cols="30" rows="2" Placeholder="Hobbies"></textarea> <br>
-<form action="/placeholder">
-<textarea name="" id="" cols="30" rows="2" Placeholder="Job Experiences"></textarea> <br>
-<textarea name="" id="" cols="30" rows="2" Placeholder="Educations"></textarea> <br>
-<input type="submit" value="Submit">
-</form>
+    <h2>Personal Information</h2>
+    <form action="" method="post" enctype="multipart/form-data">
+        <img src="/views/public/images/<?= $profileimg ?>" alt="">
+        <label for="fileToUpload" id="uploadImage">
+            <span>Select Image <span class="material-symbols-rounded">
+                    add_photo_alternate
+                </span></span>
+            <!-- hidden file input that gets replaced by the span -->
+            <input type="file" name="fileToUpload" id="fileToUpload" accept="image/*">
+        </label>
+        <input type="submit" value="Upload" name="uploadpfp">
+    </form>
+    <form action="" method="post">
```

Comment:
De action mist een url

Reply:
hoeft niet omdat die bij edit_profile.php controller een if statement doet waarin hij checkt of het een post is met de submit edituser bij regel 22

Reply:
ja zo kan het ook dan i guess :)